packages/driver/src/cypress/utils.ts

```diff
@@ -397,6 +397,7 @@ export default {
   },
 
   isPromiseLike (ret) {
-    return ret && _.isFunction(ret.then)
+    // @ts-ignore
```

Comment:
why ignore?

Reply:
Typescript linting was yelling at me....and since we are determine if something is a promise seemed okay to ignore 😅  this was the error:
```bash
src/cypress/utils.ts(400,55): error TS2339: Property 'then' does not exist on type 'object'.
src/cypress/utils.ts(400,81): error TS2339: Property 'catch' does not exist on type 'object'.
```